exercises/D-strings/exercise.js

```diff
@@ -1,3 +1,9 @@
 // Start by creating a variable `message`
+var message = "This is a string";
+var messageType = typeof message;
+console.log(message);
 
+var message = "string";
+var messageType = typeof message;
```

Comment:
be careful declaring variables with the same name several times as it will cause an error